### ISSUE TITLE
All review colour palettes to pass WCAG AA contrast standards

### ISSUE THREAD

**Issue by frederickobrien** [audioxide/data#112]
A solid 140+ of our reviews fail the AA level Web Content Accessibility Guidelines.

I want that number down to zero.
